src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -81,6 +82,10 @@ function isSmallScreen(width) {
   return  width < LARGE_SCREEN;
 }
 
+function onReviewPage(pageTitle) {
+  return pageTitle === 'Review your photo';
```

Comment:
You can tell you're on the review page by `this.props.formContext.reviewMode` being `true`.

Reply:
It looks like reviewMode is false once edit is clicked:
<img width="1372" alt="screen shot 2018-02-07 at 12 34 10 pm" src="https://user-images.githubusercontent.com/16051172/35940214-49d57180-0c03-11e8-98f0-8a5bb8e9a405.png">

Reply:
I think that's fine. If you edit, our pattern is generally that you're on the regular version of the page and it should look just like what you saw filling out the form initially.

Reply:
Thanks @jbalboni so is checking the pageTitle the best way to ensure that we are on the review page? This is used to ensure that the small screen styles are applied, even if the screen size is large, to take into consideration the narrower review page styles.

Reply:
I'm not really sure I understand why we need the small page styles if the review page is wider than the small page breakpoint we set. Should we change the small page breakpoint instead?